scripts/config/templates/nginx.conf.erb

```diff
@@ -77,6 +77,12 @@ http {
     }
   <% end %>
 
+  <% if canonical_host %>
+    if ($host != <%= canonical_host %>) {
+      return 301 $scheme://<%= canonical_host %>$request_uri;
```

Comment:
I believe `$scheme` will always resolve to `http` because the application is behind the Heroku router. Is there any way this can use the `X-Forwarded-Proto` header to detect the correct scheme?